django-server/server/admin/templates/stats.html

```diff
@@ -0,0 +1,515 @@
+{% extends 'admin/change_list.html' %}
```

Comment:
Should this file be here?

Reply:
No, this template extends one of default admin templates.

FYI usually you don't need to do any configuration except this `extends` line. For the main admin page I had to modify some code, because default Django overrides don't work there.

Reply:
I also renamed all overrided templates to end with `_override` so it's easier to see when we need something and when we don't.